tests/Feature/Console/StartCommandTest.php

```diff
@@ -104,7 +104,8 @@
     );
 
     $response->assertStatus(200);
-    $response->assertHeader('Content-Type', 'text/event-stream; charset=UTF-8');
+
+    expect(strtolower((string) $response->headers->get('Content-Type')))->toBe('text/event-stream; charset=utf-8');
```

Comment:
This test started failing on the Laravel 11 and 12 because of `UTF-8`case so just added test to test this way.